aqt/archives.py

```diff
@@ -36,18 +38,40 @@ def __init__(self, version, target, arch, os_name):
         self.arch = arch
         self.os_name = os_name
 
+    def __str__(self):
+        print(f'TargetConfig(version={self.version}, target={self.target}, '
+              f'arch={self.arch}, os_name={self.os_name}')
+
+    def __repr__(self):
+        print(f'({self.version}, {self.target}, {self.arch}, {self.os_name})')
+
```

Comment:
Adding __repr__ and __str__ for easier debugging: you can just print objects (__str__) or lists of objects (__repr__ is used)